src/main/kotlin/com/ajaxproject/telegrambot/bot/service/MonobankService.kt

```diff
@@ -0,0 +1,39 @@
+package com.ajaxproject.telegrambot.bot.service
+
+import com.ajaxproject.telegrambot.bot.dto.MonobankCurrencyExchangeResponse
+import com.ajaxproject.telegrambot.bot.properties.MonobankProperties
+import com.ajaxproject.telegrambot.bot.utils.HttpUtils
+import com.ajaxproject.telegrambot.bot.utils.JsonUtils
+import com.squareup.okhttp.Request
+import com.squareup.okhttp.Response
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Service
+
+
+@Service
+class MonobankService(
+    urlProperties: MonobankProperties,
+    val currencyExchangeService: CurrencyExchangeService,
+) {
+
+    private final val request: Request = Request.Builder().url(urlProperties.url).get().build()
```

Comment:
why final? `val` makes the property final :)
```suggestion
    private val request: Request = Request.Builder().url(urlProperties.url).get().build()
```